script.js

```diff
@@ -1,4 +1,76 @@
+// Rock Paper Scissor Game (Project 1)
+
+var currentGameMode = 'waiting for user name mode';
+var userName = '';
+
+var scoreUserWin = 0;
+var scoreCpuWin = 0;
+var scoreDraw = 0;
+
+// get random CPU pick from an array
+var cpuPick = function () {
```

Comment:
Function naming convention is to start with verb- e.g. `generateCpuPick` or something like that. https://bootcamp.rocketacademy.co/course-logistics/naming-casing-and-commenting-conventions